common/logging_extra.py

```diff
@@ -65,7 +65,7 @@ def format_dict(self, record):
 
     return record_dict
 
-  def format(self, record):
+  def format(self, record): # noqa: A003
```

Comment:
why is this ignored?

Reply:
format is a function of logging.formatter, and needs to be overridden